ansible-automation/files/my.cnf

```diff
@@ -0,0 +1,24 @@
+#
+# This group is read both both by the client and the server
+# use it for options that affect everything
+#
+[client-server]
+#
+# include all files from the config directory
+#
+!includedir /etc/my.cnf.d
+bind-address=0.0.0.0
+[client]
+socket=/home/mysql/mysql.sock
+[mysqld]
+datadir=/home/mysql/
+socket=/home/mysql/mysql.sock
+max_connections=1100
+innodb_buffer_pool_size=75G
```

Comment:
I would prefer to have this dynamic. Change this file to the Jinja template and let Ansible fill these values based on the machine hw.